src/main/java/com/glitchcog/fontificator/emoji/TypedEmojiMap.java

```diff
@@ -72,91 +69,40 @@ public LazyLoadEmoji getEmoji(String testKey, ConfigEmoji config)
             return null;
         }
 
-        LazyLoadEmoji emoji = normalMap.get(testKey);
-
-        if (emoji == null)
+        LazyLoadEmoji emoji = null;
+        if (type.canRegEx())
```

Comment:
The slashes in new badge name caused me problems because they'd get identified as regex's and then things got weird.  So I realized that the need for regex's could be sourced from the EmojiType because only twitch emoji really use regex.  This I think simplified the BTTV handling